src/open_inwoner/pdc/resources/export_resource.py

```diff
@@ -31,7 +31,7 @@ class ProductExportResource(resources.ModelResource):
     organizations = fields.Field(
         column_name="organizations",
         attribute="organizations",
-        widget=ManyToManyWidget(Organization, field="name"),
+        widget=ManyToManyWidget(Organization, field="slug"),
```

Comment:
I'm not sure about this one, because in the provided Excel file they use Organization name, not slug
Should the input for M2M fields be slugified on our end?

Reply:
The file that we have as a template on the import page has everywhere slug type names for m2m fields. I changed them as well because if we have name in the resource but slug type in the imported file we will have a problem. In my opinion it is better to have slugs because they are unique and helps with duplicates as well.